test/e2e/specs/wp-gutenberg-experimental-features-spec.js

```diff
@@ -26,6 +26,8 @@ const EXPERIMENTAL_FEATURES = {
 	'@wordpress/block-editor': [ '__experimentalBlock', '__experimentalInserterMenuExtension' ],
 	'@wordpress/date': [ '__experimentalGetSettings' ],
 	'@wordpress/interface': [ '__experimentalMainDashboardButton' ],
+	'@wordpress/components': [ '__experimentalNavigationBackButton' ],
+	'@wordpress/edit-site': [ '__experimentalMainDashboardButton' ],
```

Comment:
Out of curiosity, is there a reason we added `'@wordpress/edit-site': [ '__experimentalMainDashboardButton' ]` to the list of Calypso dependencies?

I see the use of `__experimentalMainDashboardButton`, but it's already being exported by `@wordpress/interface`.

Reply:
The [version from the edit-site package](https://github.com/WordPress/gutenberg/blob/master/packages/edit-site/src/components/main-dashboard-button/index.js) is the one we specifically set up for the navigation panel.  It is the one whose slot is added in the Nav panel, so that is the one we need to use for the fill.